src/main/java/IreneAI.java

```diff
@@ -0,0 +1,179 @@
+/**
+ * The IreneAI program implements a simple chat application that
+ * provides simple feedbacks to the user's inputs
+ *
+ * @author Chai Ming How
+ * @since 2024-02-01
+ */
+
+import javafx.scene.control.SplitPane;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class IreneAI {
+    private static final ArrayList<Task> tasks = new ArrayList<>();
+    private static final String LINE = "____________________________________________________________";
+    public static void main(String[] args) throws DukeException {
+        Scanner scanner = new Scanner(System.in);
+
+        String chatbotName = "IreneAI";
+        dividingLine(LINE);
+        System.out.println(" Hello! I'm " + chatbotName);
+        System.out.println(" What can I do for you?");
+        dividingLine(LINE);
+
+        while (true) {
+            System.out.println("You: ");
+            String userInput = scanner.nextLine();
+            String[] splitInputs = userInput.split(" ");
+            Command command = Command.getCommand(splitInputs[0]);
+            String argument = splitInputs.length > 1 ? splitInputs[1] : "";
+            dividingLine(LINE);
+
+            try {
+                switch (command) {
+                    case BYE:
```

Comment:
switch case coding standard could be done better by having format as such:
switch(command) {
case BYE..